src/odemis/acq/stream/_base.py

```diff
@@ -300,6 +320,10 @@ def det_vas(self):
     def emt_vas(self):
         return self._emt_vas
```

Comment:
This function seems like a waste of code/characters, calling the function cost more than the actual implementation of the function?

Reply:
This part is already in the code (so it's not Anders' fault).
The goal of such "property" is that it indicates the attribute is read-only. A call in Python is not very costly, but surely, if you need an extremely fast code path, this would need an optimization (or to rewrite everything in cython ;-) ).